rules/duo_rules/duo_admin_bypass_code_viewed.yml

```diff
@@ -0,0 +1,32 @@
+AnalysisType: rule
+Description: An administrator viewed the MFA bypass code for a user.
+DisplayName: Duo Admin Bypass Code Viewed
+Enabled: true
+Filename: duo_admin_bypass_code_viewed.py
+Reference: https://duo.com/docs/adminapi
+Runbook: 'Confirm this behavior is authorized: The security of your Duo application is tied to the security of your secret key (skey). Secure it as you would any sensitive credential. You should not share it with unauthorized individuals or email it to anyone under any circumstances!'
```

Comment:
This does not need quotes ( though no particular harm in having them )

Reply:
fixed in new commit!